.circleci/config.yml

```diff
@@ -34,31 +34,21 @@ jobs:
           name: Lint
           command: |
             yarn lint
-  deploy:
-    working_directory: ~/pennlabs
-    docker:
-      - image: pennlabs/labs-helm
-    steps:
-      - checkout
-      - run:
-          name: Deploy to k8s
-          command: /deploy.sh
 
 workflows:
   version: 2
   build-and-deploy:
     jobs:
       - test
       - docker/publish:
-          name: publish
           requires:
             - test
           image: pennlabs/pennbasics
-          tag: $CIRCLE_SHA1
+          tag: $CIRCLE_SHA1,latest
```

Comment:
This probably should be wrapped in quotation marks

Reply:
why tho